static/src/stylesheets/module/commercial/_adverts-soulmates.scss

```diff
@@ -85,8 +85,42 @@
     }
 }
 
+.advert-blended--soulmates {
+    background: #f5f3e7;
```

Comment:
Should this go from our palette? If not should we put it into variable in top of the file or to some sort of banding section?